src/Arcus.BackgroundJobs.AzureActiveDirectory/AzureApplication.cs

```diff
@@ -4,19 +4,19 @@
 namespace Arcus.BackgroundJobs.AzureActiveDirectory
 {
     /// <summary>
-    /// Represents an <see cref="ApplicationWithExpiredAndAboutToExpireSecrets"/> from Azure Active Directory.
+    /// Represents an <see cref="AzureApplication"/> from Azure Active Directory.
     /// </summary>
-    public class ApplicationWithExpiredAndAboutToExpireSecrets
+    public class AzureApplication
```

Comment:
Renamed this, as the model contained '...And...;

Reply:
Let me know if you don't agree with this.

Reply:
Fine by me, `AzureApplication` is descriptive enough and I was already doubting if `ApplicationWithExpiredAndAboutToExpireSecrets` might be a bit too long.